src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -264,6 +264,36 @@ protected virtual Task OnPostActorMethodAsync(ActorMethodContext actorMethodCont
                 Ttl = ttl
             });
         }
+        
+        /// <summary>
+        /// Registers a reminder with the actor.
+        /// </summary>
+        /// <param name="reminderName">The name of the reminder to register. The name must be unique per actor.</param>
+        /// <param name="state">User state passed to the reminder invocation.</param>
+        /// <param name="dueTime">The amount of time to delay before invoking the reminder for the first time. Specify negative one (-1) milliseconds to disable invocation. Specify zero (0) to invoke the reminder immediately after registration.</param>
+        /// <param name="period">
+        /// The time interval between reminder invocations after the first invocation. Specify negative one (-1) milliseconds to disable periodic invocation.
```

Comment:
I know you got this from above, but is -1 even a valid period? I think an empty period is actually correct and -1 may be invalid.

https://docs.dapr.io/reference/api/actors_api/#create-actor-reminder

Reply:
Just saw that we're accepting values > 0 for repetitions. So, getting rid of that statement.